### PR TITLE
DATA: Created IGenericRepository Interface.

### PR DESCRIPTION
**Pull request by khabibulladev**
Created IGenericRepository Interface Inside IRepositories Folder.

### CODE REVIEW

**Comment by khabibulladev** [comment]
Closes #29

File: BelissimoCloneWPF.Data/IRepositories/IGenericRepository.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Linq.Expressions;
+
+namespace BelissimoCloneWPF.Data.IRepositories
+{
+    public interface IGenericRepository<T> where T : class
+    {
```
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Data/IRepositories/IGenericRepository.cs:6]: Make Generic work only with Auditable

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

**Comment by khabibulladev** [comment]
Made Generic work only with Auditable

**Comment by MuhammadKasimov** [comment]
Man what are you doing!?

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]
It had to be 
public interface IGenericRepository<T> where T : Auditable

**Pull request merged**